querybook/webapp/components/DataTableNavigator/SchemaTableView/SchemaTableView.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useRef } from 'react';
+import styled from 'styled-components';
+import { useDispatch, useSelector } from 'react-redux';
+import { Dispatch, IStoreState } from 'redux/store/types';
+import type { ITableSearchResult } from 'redux/dataTableSearch/types';
+
+import {
+    searchSchemas,
+    searchTableBySchema,
+} from 'redux/dataTableSearch/action';
+import { SchemaTableItem } from './Item';
+
+const CategoriesList = styled.div`
+    height: 50vh;
+    overflow-y: auto;
+`;
+
+const IntersectionElement = styled.div`
+    width: 100%;
+    height: 1px;
+`;
+
+function prepareSchemaNames(
+    tables: ITableSearchResult[]
+): ITableSearchResult[] {
+    if (!tables) {
+        return [];
+    }
+
+    return tables.map((table) => ({
+        ...table,
+        full_name: table.full_name.split('.').slice(1).join('.'),
```

Comment:
if you want to use table name you can do `full_name: table.name`

Reply:
@czgu Now I can't see this field in response. Could you help me with it? Should I get this extra field from elasticsearch? Thank you